### ISSUE TITLE
Coloured Redstone not connecting to repeaters in particular setups

### ISSUE THREAD

**Issue by hanakocz** [ganymedes01/Gany-s-Surface#79]
Hello,
I found bug with your coloured redstone. It works really nice, but in some setups it does not connect to redstone repeaters nor other thingies (comparator).
It does when the connecting "red"stone is branched on the block it is connected at. 
Version 1-12.7 (latest as I know)
I think pictures will say more than words:

![2016-01-18_14 20 41](https://cloud.githubusercontent.com/assets/8724169/12392538/3700be38-bdf0-11e5-9031-5dfcf3a85e61.png)
This one: Look that when straight, green one will conect, but when branched, it won't. Standard redstone hasn't this issue.
![2016-01-18_14 24 49](https://cloud.githubusercontent.com/assets/8724169/12392539/370217ec-bdf0-11e5-98f8-9826c8718d30.png)
Comparator is disconnected, when the input greenstone is branched. (but only on side)
![2016-01-18_14 24 54](https://cloud.githubusercontent.com/assets/8724169/12392537/36fe9f54-bdf0-11e5-8565-fe06019ddf1e.png)

---

One more strange thing:
When it just branches , it won't connect as I showed above, but when it just branches because of other redstone accepting block, it works as it should: (same with repeater)
![2016-01-18_14 35 02](https://cloud.githubusercontent.com/assets/8724169/12392671/ed2e7952-bdf0-11e5-815d-1c2b71168ab1.png)
![2016-01-18_14 35 15](https://cloud.githubusercontent.com/assets/8724169/12392672/ed3071ee-bdf0-11e5-82c1-596727640f55.png)

**Comment by buttilda** [comment]
Fixed the rendering, but the fact that the signal doesn't go through seems to be because of vanilla's hardcoded it to only work with the redstone wires ಠ_ಠ
Not sure what can be done, but I'll keep looking into it :3

**Comment by hanakocz** [comment]
Well, it goes through, but only when there is straight line into the repeater. I think you already got it, but one more fancy picture :)))
![2016-01-18_19 02 10](https://cloud.githubusercontent.com/assets/8724169/12398790/2aee4770-be16-11e5-88ec-f3ead7389094.png)

**Comment by buttilda** [comment]
Yeah, for some reason unless it's a "forced" connection (by making it a straight line) the repeaters/comparators won't pick up the signal change
